frontend/.env.development

```diff
@@ -5,5 +5,6 @@ REACT_APP_DECODER=$DOMAIN/decoder
 REACT_APP_DEVICE=$DOMAIN/device
 REACT_APP_STREAM=$DOMAIN/stream
 REACT_APP_TOKEN=$DOMAIN/auth/token
-REACT_APP_LOG=$DOMAIN/log
 REACT_APP_SIGNUP=$DOMAIN/user/sign-up
+REACT_APP_LOG=$DOMAIN/log
+REACT_APP_STREAMLOGS=$DOMAIN/log/stream
```

Comment:
Stick to the singular with `REACT_APP_STREAMLOG`

Reply:
1b904e1